pennylane/gradients/parameter_shift.py

```diff
@@ -142,11 +139,11 @@ def _get_operation_recipe(tape, t_idx, shifts, order=1):
         ) from e
 
     # Create shift rule from frequencies with given shifts
-    coeffs, shifts = qml.gradients.generate_shift_rule(frequencies, shifts=shifts, order=order)
+    coeffs, shifts = qml.gradients.generate_shift_rule(frequencies, shifts=shifts, order=order).T

```

Comment:
Question: are there any docstring examples that have `coeffs, shifts = ...` that we need to modify to take into account the transpose?

Reply:
Oh, very good call, I'll check!

Reply:
I didn't find any :)

Reply:
💯